### DISCUSSION TITLE
The next release (6.2.66) and betas (6.2.65b) of Sympa

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1624] [Release works]
I think sympa-community/sympa#1206 and sympa-community/sympa#1207 may be candidates of backporting by the packagers.  Or, would we be better to release a new version including this two fixes right now?

**Comment by racke** [comment]
Thanks for the heads-up. I think we can simply add the patches to the package, no need to release right now IMHO.

**Comment by ikedas** [comment]
Thanks for response.  If no objection, we'd like not to release patched version.

BTW I think one of highlights on the next regular version will be sympa-community/sympa#1186.  I look forward to help of you all in debugging and so on.

**Comment by ikedas** [comment]
I propose to release a beta in the next week.

**Comment by ikedas** [comment]
Done!

**Comment by ikedas** [comment]
I propose 6.2.65b.2 will be out in this weekend.

To know what the next beta will contain, see the [Milestone 6.2.66](https://github.com/sympa-community/sympa/milestone/18).

**Comment by ikedas** [comment]
Ah it has been done in 30 Aug.

**Comment by ikedas** [comment]
I was on vacation last week.

If there are no objections, Let's release the stable, 6.2.66, on Wed 29 or Thu 30.

**Comment by racke** [comment]
Works for me :+1:

**Comment by ikedas** [comment]
Done!

listes.renater.fr server is busy now.
``` code
$ telnet mxb.relay.renater.fr 25
Trying 194.214.201.9...
Connected to mxb.relay.renater.fr.
Escape character is '^]'.
421 4.3.2 All server ports are busy
Connection closed by foreign host.
$
```

Distribution of release announcement will be delayed.

**Comment by racke** [comment]
Thanks for the release!
